src/components/commons/CommonBookCard.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Card,
+  CardMedia,
+  CardContent,
+  Typography,
+  Rating,
+  Box,
+  SxProps,
+  Stack,
+} from '@mui/material';
+import PeopleIcon from '@mui/icons-material/People'; // 사람 아이콘 추가
+
+interface CommonBookCardProps {
+  image: string; // 이미지
+  title?: string; // 제목
+  author?: string; // 저자
+  customerReviewRank?: number; // 0~10점 값
+  priceStandard?: number; // 정가 가격
+  ratingCount?: number; // 평가 인원 수
+  onClick?: () => void; // 상세페이지 이동 을 위한 onClick 함수 생성
+  sx?: SxProps; // MUI의 sx 스타일 속성
+}
+
+const CommonBookCard = ({
+  image,
+  title,
+  author,
+  customerReviewRank,
+  priceStandard,
+  ratingCount,
+  sx,
+  onClick,
+}: CommonBookCardProps): JSX.Element => {
+  const convertedRating = customerReviewRank ? customerReviewRank / 2 : 0; // 10점 만점 -> 5점 만점으로 변환
+  const displayRating = Number.isInteger(convertedRating)
+    ? convertedRating.toFixed(0) // 정수일 경우 소수점 제거
+    : convertedRating.toFixed(1); // 소수점 1자리 유지
+
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        cursor: 'pointer',
+        ...sx,
+      }}
+      onClick={onClick}
+    >
+      <CardMedia
+        component="img"
+        image={image}
+        alt={title || '책 이미지'}
+        sx={{
+          height: 200, // 고정된 높이
+          objectFit: 'fill', // 이미지를 cover 방식으로 표시
+          objectPosition: 'center', // 이미지 중심 정렬
+          borderRadius: '8px 8px 0 0', // 상단 좌우 둥근 테두리
+        }}
+      />
+      <CardContent>
+        {title && (
+          <Typography
+            variant="body1"
+            fontWeight="bold"
+            gutterBottom
+            sx={{ textOverflow: 'ellipsis' }}
+          >
+            {title}
+          </Typography>
+        )}
+        {author && (
+          <Typography variant="body2" sx={{ color: 'grey.700' }}>
+            {author}
+          </Typography>
+        )}
```

Comment:
```suggestion
        {title && (
          <Typography
            variant="body1"
            fontWeight="bold"
            gutterBottom
            sx={{
              overflow: 'hidden',
              textOverflow: 'ellipsis',
              display: '-webkit-box',
              WebkitBoxOrient: 'vertical',
              WebkitLineClamp: 2,
            }}
          >
            {title}
          </Typography>
        )}
        {author && (
          <Typography
            variant="body2"
            sx={{
              color: 'grey.700',
              overflow: 'hidden',
              textOverflow: 'ellipsis',
              display: '-webkit-box',
              WebkitBoxOrient: 'vertical',
              WebkitLineClamp: 1,
            }}
          >
```

디자인 관련이라 반영 안 해주셔도 되는데, 혹시나 더 괜찮아 보인다면 반영해주셔용!
제목이랑 글쓴이가 너무 길어지면 카드도 따라서 너무 길어보이는 것 같아서, 제목은 2줄, 글쓴이는 1줄로 제한해봤습니다! 아래 이미지 참고해주세요!

<img width="892" alt="image" src="https://github.com/user-attachments/assets/8c2579bc-82ad-421c-bee4-69edf592bfce" />

Reply:
필요없는 컴포넌트 삭제 하겠습니다.
줄임으로 피드백 수정 하겠습니다!